From 2cd5f9db47560c1d2495434a620f0b79a44c6fa1 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Fri, 29 Sep 2023 14:27:30 -0500
Subject: [PATCH] Fix version generation during reconfigure

If we don't do this then an `-unknown` is appended to the version string during %reconfigure since the version detection script is designed to be ran in a git repo
---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 4e1665b9..a1758087 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -274,7 +274,7 @@ if [ "$myhost" = "find-version" ]; then
       rvd="0"
     fi
 
-    echo "$package-$vers$tmp:$beta:$ingit:$vers$tmp:$vers:$tmp:$rev:$rvd:$cid:"
+    echo "$package-$vers:$beta:$ingit:$vers:$vers:$rev:$rvd:"
     exit 0
 fi
 # **** end FIND VERSION ****
-- 
2.42.0

